lib/instrumentation/modules/cassandra-driver.js

```diff
@@ -0,0 +1,50 @@
+'use strict'
+
+/**
+ * TODO: Support batch queries
+ */
+
+var semver = require('semver')
+var sqlSummary = require('sql-summary')
+
+var shimmer = require('../shimmer')
+
+module.exports = function (cassandra, agent, version, enabled) {
+  if (!enabled) return cassandra
```

Comment:
Does cassandra have a callback queue we need to patch even if disabled?

Reply:
Nope. There's no pooling or callback queue in the driver.